src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
```

Comment:
You can set the export here if you want 
```suggestion
export default function ActivityList({activities, onHandleDelete}) {
```